src/katgpucbf/fgpu/engine.py

```diff
@@ -1093,7 +1093,7 @@ class Engine(aiokatcp.DeviceServer):
     src_packet_samples
         The number of samples per digitiser packet.
     src_buffer
-        The size of the network receive buffer (per polarisation).
+        The size of the network receive buffer.
```

Comment:
The actual behaviour change occurred when stream-per-pol was replaced by a single stream, but the documentation was missed at the time.